src/index.js

```diff
@@ -74,6 +75,25 @@ const optionConfig = {
         short: "r",
         multi: true,
         help: "Include translated resource files in the output files such that they can be loaded with ilib-resbundle. The resource files should come from ilib's loctool or other such localization tool which produces a set of translated resource files. VAL is the path to the root of a resource file tree."
+    },
+    legacyilib: {
+        short: "o",
+        flag: true,
+        help: "The flag to indicate assembe the legacy version of ilib."
+    },
+    ilibPath: {
+        short: "i",
+        help: "Specify the location where the legacy versin of ilib is installed."
+    },
+    ilibincPath: {
+        short: "f",
+        "default": "./src/ilib-assemble-inc.js",
+        help: "Specify name of Javascript file to process."
```

Comment:
-> "Specify the name of the Javascript inc file to process for legacy ilib assembly."